README.md

```diff
@@ -13,16 +13,19 @@ and it will take care of deploying NSX integration components, and precisely:
 * The nsx-ncp-bootstrap daemonset
 * The nsx-node-agent daemonset
 
-For Openshift 4 clusters, the nsx-container-plugin operator monitors the network
-CR in the config.openshift.io namespace to update the container network CIDRs
-used by NCP.
-
-The nsx-container-plugin operator also monitors a dedicated ConfigMap, applies
-changes to NCP and nsx-node-agent configuration, and restart the relevant pods
+The nsx-container-plugin operator monitors a dedicated ConfigMap, applies changes
+to NCP and nsx-node-agent configuration, and creates/restarts the relevant pods
 so that the relevant configuration changes are picked up.
 
-In addition, the nsx-container-plugin operator monitors nodes ensuring the
-corresponding NSX logical port is enabled as a container host logical port.
+The nsx-container-plugin operator also monitors the nsx-node-agent status and
+updates the network status on relevant nodes.
+
+In addition, the nsx-container-plugin operator is able to monitor nodes ensuring
+the corresponding NSX logical port is enabled as a container host logical port.
+
+For Openshift 4 clusters, the nsx-container-plugin operator especially monitors
+the network CR in the config.openshift.io namespace to update the container network
```

Comment:
It is not `config.openshift.io namespace`, how about Network.config.openshift.io CR?

Reply:
Good eyes, it's a historical problem... Fixed, thanks!